namelists/echam/6.3.05p2-wiso/PI-CTRL/namelist.echam

```diff
@@ -61,11 +61,19 @@
 /
 &set_stream
     stream = 'jsbach'
+<<<<<<< HEAD
+    lpost = 0
+/
+&set_stream
+    stream = 'land'
+    lpost = 0
+=======
     lpost = 1
 /
 &set_stream
     stream = 'land'
     lpost = 1
+>>>>>>> 99e58af12b79649a45868fd0df4b22525c66235b
```

Comment:
This will break the namelist. You are probably wondering where does this come from: it's a git unresolved conflict that you have probably added (`git add`). What it means is that git does not know which of the blocks in between the `<</==/>>` markers to choose. To solve this issue you have to choose one of them and delete the `<<< HEAD`, `=====`, and `>>>>`.

This is however, the default namelist for ECHAM+WISO. I guess there is no harm of having an additional stream here right?